src/components/apiDelegation/reusables/OverviewPageContent/OverviewPageContent.tsx

```diff
@@ -152,9 +152,14 @@ export const OverviewPageContent = ({
             </Button>
           </div>
         )}
-        <Panel title={t('api_delegation.card_title')}>
-          {t('api_delegation.api_panel_content')}
-        </Panel>
+        <div className={classes.infoPanelWrapper}>
+          <Panel
+            title={t('api_delegation.card_title')}
+            forceMobileLayout
+          >
+            {t('api_delegation.api_panel_content')}
+          </Panel>
+        </div>
         <div className={classes.pageContentContainer}>
           <h2 className={classes.apiSubheading}>{accessesHeader}</h2>
           <div className={classes.editButton}>
```

Comment:
(Not the right line of code, but I'm not allowed to comment where it would be relevant)

![image](https://user-images.githubusercontent.com/22012997/216379039-a35265c8-fb79-4854-9802-38f6d417c106.png)

Hm... this looks a bit tight.. 🤔
The design of this issue tells us one way to get some space: by dropping the text stating the number of APIs connected to the org. You can put it in the subtitle, and when that too disappears on the smallest screens, then so be it.

Reply:
Another thing we should look at are these add-buttons.
Since space is so sparce, I suggest (and I know UX agrees with me on this) we should probably move this button inside the accordion (top center inside its content).
Should this be the case for all screen sizes? Maybe it should. But that discussion is probably one we should have all three of us 🙂

Reply:
I Runde 2 har jeg jeg ikke begynt på Accordion-delen i hele tatt. Men fine innspill ;)

Reply:
Åja... Det visste jeg ikke. Jeg ville vel egentlig forventet at når PRen kom inn for review så betydde det at utviklereren mente tasken var ferdig løst... 😅